boring-sys/Cargo.toml

```diff
@@ -57,9 +57,15 @@ features = ["rpk", "pq-experimental", "underscore-wildcards"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-# Use a FIPS-validated version of boringssl.
+# Compile boringssl using the FIPS build flag if building boringssl from
+# scratch.
```

Comment:
What is the "FIPS build flag"? Are you referring to a way to build boringSSL itself? If so, perhaps add a reference to the documentation.

Reply:
Yes it's a boringssl build flag, I added a link